.github/workflows/Gauth-CD.yml

```
@@ -0,0 +1,60 @@
+name: Gauth Server CD
+
+on:
+  push:
+    branches: [ main ]
+
+env:
+  S3_BUCKET_NAME: gauth-deploy-bucket
+  PROJECT_NAME: Gauth-Server
+
+jobs:
+  build:
+    name: CD
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'zulu'
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x ./gradlew
+        shell: bash
+
+      - name: Create application.yml
+        run: |
+          touch ./src/main/resources/application.yml
+          echo "${{ secrets.APPLICATION_YML }}" > ./src/main/resources/application.yml
+        shell: bash
+
+      - name: Build with Gradle
+        run: ./gradlew clean -i -x test
+        shell: bash
+
+      - name: Make zip file
+        run: zip -r ./$GITHUB_SHA.zip .
+        shell: bash
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.GAUTH_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.GAUTH_SECRET_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Upload to S3
+        run: aws s3 cp --region ap-northeast-2 ./$GITHUB_SHA.zip s3://$S3_BUCKET_NAME/$PROJECT_NAME/$GITHUB_SHA.zip
```

```suggestion
        run: aws s3 cp --region ap-northeast-2 ./$GITHUB_SHA.zip s3://$S3_BUCKET_NAME/$PROJECT_NAME/$GITHUB_SHA.zip
```
secrets에 있는 리전 쓰는게 더 나을거같아용

---

아하, 그게 더 좋을것 같네요! 수정하겠습니다!